trio/_core/_run.py

```diff
@@ -2433,17 +2465,17 @@ def unrolled_run(
 
 
 class _TaskStatusIgnored:
-    def __repr__(self):
+    def __repr__(self) -> str:
         return "TASK_STATUS_IGNORED"
 
-    def started(self, value=None):
+    def started(self, value: Any = None):
```

Comment:
Pedantic note: `object` here? (I realize it doesn't change anything at all...)

Also return type